tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/PortalServerBehavior.java

```diff
@@ -908,10 +962,36 @@ public void startModule( IModule[] modules, IProgressMonitor monitor ) throws Co
         startOrStopModules( modules, "start", monitor );
     }
 
-    @Override
-    public void stopModule( IModule[] modules, IProgressMonitor monitor ) throws CoreException
+    public void startModules()
```

Comment:
Understand, can You or Andy send this change as a separate PR so we can keep them separate?  thanks